pkg/minikube/constants/constants.go

```diff
@@ -65,9 +68,29 @@ const (
 	DefaultVMDriver = "virtualbox"
 )
 
+var DefaultKubernetesVersion = version.Get().GitVersion
+
 const (
 	RemoteLocalKubeErrPath = "/var/lib/localkube/localkube.err"
 	RemoteLocalKubeOutPath = "/var/lib/localkube/localkube.out"
 )
 
 var ConfigFilePath = MakeMiniPath("config")
+
+var localkubeDownloadURLPrefix = "https://storage.googleapis.com/minikube-test/k8sReleases/"
```

Comment:
Is this URL going to change? What is the process for new localkube builds when new releases on kubernetes come out?